launchdarkly-server-sdk-1.0-0.rockspec

```diff
@@ -13,18 +15,33 @@ dependencies = {
 
 external_dependencies = {
     LD = {
-        header = "launchdarkly/api.h"
+        header = "launchdarkly/server_side/bindings/c/sdk.h",
+        library = "launchdarkly-cpp-server"
+    },
```

Comment:
This is awkward. The idea is that luarocks can detect if we have the proper build dependencies installed and warn if they aren't. 

In this case, it'll look for the `boost_json` library in `BOOST_DIR` (passed in on command line). This totally isn't portable to Mac or Windows, so luarocks also provides platform specific formatting - it'll add `lib` or `.so` or whatever. But boost has an even more complicated format with the `-mt-x64.so.1.81.0` so I don't think it works.

TODO: Is there a better way?